Sprint-3/implement/get-angle-type.test.js

```diff
@@ -0,0 +1,48 @@
+const getAngleType = require('./path-to-your-function-file');
```

Comment:
This might be a leftover. does the test even run now?  because path-to-your-function-file does not exist in the directory.

Reply:
You are right. I've changed the path accordingly.
const getAngleType = require('./get-angle-type.test.js');